src/components/screens/questionnair-new-quiz-screen.tsx

```diff
@@ -0,0 +1,160 @@
+import { Button, Card, CardActions, CardContent, Grid, TextField, Typography } from "@mui/material";
+import DeleteForeverIcon from "@mui/icons-material/DeleteForever";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import NewQuestionCard from "./question-form";
+import { KeyboardReturn } from "@mui/icons-material";
+import UserRoleUtils from "src/utils/user-role-utils";
+
+/**
+ * New Questionnaire Card component
+ * Includes a card (question-form.tsx) to create a new question
+ */
+const NewQuizCard = () => {
+  const adminMode = UserRoleUtils.adminMode();
+  /**
+   * State and function for the questionnaire title change
+   */
+  const [questionnaireTitle, setQuestionnaireTitle] = useState("");
+
+  const handleQuestionTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuestionnaireTitle(event.target.value);
+  };
+  /**
+   * State and functions to handle the questions and options
+   * @returns {object}
+   */
+  const [questions, setQuestions] = useState<
+    { questionText: string; options: { label: string; value: boolean }[] }[]
+  >([]);
+
+  const handleAddQuestionSubmit = (
+    questionText: string,
+    options: { label: string; value: boolean }[]
+  ) => {
+    setQuestions((prevQuestions) => [...prevQuestions, { questionText, options }]);
+  };
+
+  const handleDeleteQuestion = (index: number) => {
+    const updatedQuestions = [...questions];
+    updatedQuestions.splice(index, 1);
+    setQuestions(updatedQuestions);
+  };
+  /**
+   * This should save the question title and questions to DB.
+   * TODO
```

Comment:
Is this comment TODO:

When leaving TODO: comments use the stntax TODO: and then install the TODO highlight VScode extensions for helpful highlighting